backend/loernwerkUtilities.ts

```diff
@@ -61,6 +63,47 @@ export function requireBody(...data: string[]) {
     };
 }
 
+/**
+ * Middleware for specifying the request attributes needed by the H5P library.
+ * @param req Request object
+ * @param _ Response object
+ * @param next Next handler function
+ */
+export async function buildH5PRequest(
+    req: Request,
+    _: Response,
```

Comment:
_ is a really bad name

Where is it even used?

Reply:
It's never used, but I still got to keep it there because I need the third argument - `_` was chosen because it is infact never used

Reply:
still think we should name it res to stay consistent and then say in a comment that it is never used